### ISSUE TITLE
JAWS on Firefox doesn't read <select> value containing comments

### ISSUE THREAD

**Issue by wkeese** [FreedomScientific/VFO-standards-support#470]
### Summary

When a `<select>`'s selected option contains comments, JAWS on Firefox won't read the value.

For example:

```
<select>
    <option value="1" selected=""><!---->1<!----></option>
 </select>
```

This is an issue because lit-html inserts comments everywhere.  

### Expected result

Should announce "combobox 1".

### Actual result

Just announces "combobox"

### Example

See the **second** `<select>` in https://web-components.carbondesignsystem.com/iframe.html?id=components-pagination--default-story for a working example.  (Note that the first `<select>`  is working because it has no comments.)

#### JAWS version and build number

JAWS 2019, JAWS 2020, JAWS 2021 beta
 
#### Operating System and version

Windows 10

#### Browser and version:

Firefox 83

**Comment by JAWS-test** [comment]
This is not a bug in JAWS, but in Firefox as you can see in the Accessibility tab of Firefox: Firefox does not transmit a value to the Accessibility API. The problem does not occur with Chrome, but with Firefox+NVDA. The problem occurs in Firefox with any navigation method and when changing list items in closed state, but not when changing them in open state. Please post the error here: https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox

**Comment by wkeese** [comment]
OK thanks.  I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1678534.
